### DISCUSSION TITLE
How to update the firmware

### DISCUSSION THREAD

**Discussion by brendenric** [radensb/eZ_Navi-Delete#7] [General]
Hi Ryan,

Can you give an easy step by step instruction of how to update the firmware for those of us with an earlier version of the board?  I've been reading through the USB load process but in terms of getting your latest programing onto the board, I'm not sure how it is done.

Thank you! 

  Brenden

**Comment by radensb** [comment]
The build environment needs to be setup to build/program the code, whether you are programming a board for the first time, or updating. You can follow the steps in the bring up section of the Wiki for either case as the updating the code is the same process.
https://github.com/radensb/eZ_Navi-Delete/wiki/eZ-Navi-Delete-Bring-Up

There is also a link to a video that illustrates all the steps needed:
https://www.youtube.com/watch?v=9-2JsU4G_ls

In general (once the build env is setup), you just need to update your local copy of the repo (if you have it checked out) or download the current repo head ZIP file (as the instructions direct) to get the current code files. Then, connect the board via USB, select the proper programming settings in the IDE, and build/upload the code to the board.

Let me know if there are any issues!
